realm/realm-library/src/main/java/io/realm/RealmFieldType.java

```diff
@@ -45,15 +48,29 @@
     DOUBLE(10),
     OBJECT(12),
     LIST(13),
-    LINKING_OBJECTS(14);
+    LINKING_OBJECTS(14),
+
+    @SuppressWarnings("PointlessArithmeticExpression")
+    INTEGER_LIST(LIST_OFFSET + 0),
```

Comment:
What's the `BYTE_LIST`? Isn't it the same with `BINARY(4)`?

And `BLOB_LIST ` means we need to support `RealmList<byte[]>`, right?